cmd/kubeadm/app/master/manifests_test.go

```diff
@@ -539,7 +538,7 @@ func TestGetAPIServerCommand(t *testing.T) {
 	}
 
 	for _, rt := range tests {
-		actual := getAPIServerCommand(rt.cfg, false)
+		actual := getAPIServerCommand(rt.cfg, false, version.MustParseSemantic("v1.6"))
```

Comment:
How about a small test for 1.7 that asserts that the proxy-client flags get added?